app/src/main/java/org/p2p/wallet/home/ui/main/bottomsheet/HomeActionsBottomSheet.kt

```diff
@@ -55,12 +53,6 @@ class HomeActionsBottomSheet :
         expandToFitAllContent()
```

Comment:
remove it from here as lon as it is called in BaseMvpBottomSheet

Reply:
it's not called in BaseMvpBottomSheet